pkg/sql/catalog/systemschema/system.go

```diff
@@ -675,6 +675,16 @@ CREATE TABLE system.external_connections (
 	CONSTRAINT "primary" PRIMARY KEY (connection_name),
 	FAMILY "primary" (connection_name, created, updated, connection_type, connection_details)
 );`
+
+	SystemJobInfoTableSchema = `
+CREATE TABLE system.job_info (
+	job_id INT8 NOT NULL,
+	info_key BYTES NOT NULL,
+	written TIMESTAMPTZ NOT NULL DEFAULT now(),
+	value BYTES,
+	CONSTRAINT "primary" PRIMARY KEY (job_id, info_key, written DESC),
```

Comment:
why is `written` part of the primary key, given that the accessor helpers (in commit 3) look up by (job_id, info_key)?

I guess I'll need to read the implementation to see how we handle the case when there are >1 rows with the same (job_id, info_key) but different 'written' time.